Jaguar entry point

andrioli opened the pull request:
Create Jaguar entry point class. Still work-in-progress, but it will be
useful to start implementing the test listeners and providers.

henriquelemos0 commented:
JDK 10 is not supported ??? In which world do they live ???
Ok then, those are more than enough to start.

andrioli committed:
Jaguar entry point

Create Jaguar entry point class. Still work-in-progress, but it will be
useful to start implementing the test listeners and providers.
